server/apps/microservices/src/processors/metadata-extraction.processor.ts

```diff
@@ -60,8 +60,8 @@ export class MetadataExtractionProcessor {
       newExif.make = exifData['Make'] || null;
       newExif.model = exifData['Model'] || null;
       newExif.imageName = path.parse(fileName).name || null;
-      newExif.exifImageHeight = exifData['ExifImageHeight'] || null;
-      newExif.exifImageWidth = exifData['ExifImageWidth'] || null;
+      newExif.exifImageHeight = exifData['ExifImageHeight'] || exifData['ImageHeight'] || null;
```

Comment:
is the `ImageHeight` come from TIFF file only?

Reply:
Not sure for more types in feature but only for TIFF now. I don't know why `exifr` exposed `ImageHeight` and `ImageWidth` fields instead of `ExifImageHeight`. I guest it come from TIFF meta data.

Reply:
That makes sense